.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,40 @@
+name: 'tests'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: 'ubuntu-latest'
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
Currently, we have renovate updating these versions. In this setup, will we lose this convenience? I wouldn't like losing it.

Reply:
See: https://github.com/standard/eslint-config-standard-with-typescript/pull/533#issuecomment-785785803

Reply:
As I responded there—I wouldn't like to lose this convenience.

Reply:
Hmmm, sorry if I'm missing something, but if renovate is updating these versions won't we stop testing on a specific version of Node.js without cutting a breaking change? As I see it, we should only remove older Node.js versions here when we drop support for them by changing `"engines"` in `package.json`, and release a new major version? 🤔

Reply:
If renovate removes a Node.js version, that PR should not be automerged. It should be manually merged, having its commit message changed to include the removal of support. I don't remember whether renovate currently auto-merges these particular PRs. And I don't remember the last time this had happened.

Reply:
Thanks to @voxpelli note in https://github.com/standard/eslint-config-standard-with-typescript/issues/598#issuecomment-871624204.

We can now use `lts/*` for `node-version`, so we still support all the versions of Node.js.